image-inspector.go

```diff
@@ -126,9 +164,16 @@ func main() {
 	if _, err := client.InspectImage(*image); err != nil {
 		log.Printf("Pulling image %s", *image)
 		imagePullOption := docker.PullImageOptions{Repository: *image}
-		imagePullAuth := docker.AuthConfiguration{} // TODO: support authentication
-		if err := client.PullImage(imagePullOption, imagePullAuth); err != nil {
-			log.Fatalf("Unable to pull docker image: %v\n", err)
+		imagePullAuths := getAuthConfigs(dockercfg, username, password_file)
+		// Try all the possible auth's from the config file
+		var auth_err error
```

Comment:
@enoodle why changing naming conventions? We're using camelcase instead of underscores, I would expected this to be something like `authError`.